src/pyvale/examples/blender/ex1_1_blenderscene.py

```diff
@@ -0,0 +1,127 @@
+                                 focal_length=15.0)
+    camera = pyvale.BlenderScene.add_camera(cam_data)
+
+    # The camera can be moved and rotated this:
+    camera.location = (0, 0, 410)
+    camera.rotation_euler = (0, 0, 0) # NOTE: The default is an XYZ Euler angle
+
+
+    # Add the light
+    light_data = pyvale.BlenderLightData(type=pyvale.BlenderLightType.POINT,
+                                         pos_world=(0, 0, 400),
+                                         rot_world=Rotation.from_euler("xyz",
+                                                                       [0, 0, 0]),
+                                         energy=1)
+    light = pyvale.BlenderScene.add_light(light_data)
+
+    # The light can also be moved and rotated:
+    light.location = (0, 0, 410)
+    light.rotation_euler = (0, 0, 0)
+
+    # Apply the speckle pattern
+    material_data = pyvale.BlenderMaterialData()
+    speckle_path = pyvale.DataSet.dic_pattern_5mpx_path()
+    # NOTE: If you wish to use a bigger camera, you will need to generate a
+    # bigger speckle pattern generator
+
+    mm_px_resolution = pyvale.CameraTools.calculate_mm_px_resolution(cam_data)
+    pyvale.BlenderScene.add_speckle(part=part,
+                                    speckle_path=speckle_path,
+                                    mat_data=material_data,
+                                    mm_px_resolution=mm_px_resolution)
+
+    # Rendering image
+    # --------------------------------------------------------------------------
+    # Set this to True to render image of the current scene
+    render_opts = True
+    if render_opts:
+        save_dir = Path.cwd() / "blenderimages"
+        # NOTE: If no save directory is specified, this is where the images will
+        # be saved
+        save_name = "ex1_1"
+        render_data = pyvale.RenderData(cam_data=cam_data,
+                                        save_dir=save_dir,
+                                        save_name=save_name,
+                                        threads=8)
+        # NOTE: The number of threads used to render the images is set within
+        # RenderData, it is defaulted to 4 threads
+
+        pyvale.BlenderScene.render_single_image(bounce_image=False,
+                                                render_data=render_data)
+        # NOTE: If bounce_image is set to True, the image will be saved to disk,
+        # converted to an array, deleted and the image array will be returned.
+
+        print()
+        print(80*"-")
+        print("Save directory of the image:", render_data.save_dir)
+        print(80*"-")
+        print()
+
+    # Save Blender file
+    # --------------------------------------------------------------------------
+    # The file that will be saved is a Blender project file. This can be opened
+    # with the Blender GUI to view the scene.
+    blender_path = Path.cwd() / "blenderfiles/ex1_1.blend"
+    pyvale.BlenderTools.save_blender_file(blender_path)
```

Comment:
I'm getting an error here:

/path/to/pyvale/src/pyvale/data/blender_files/ex1_1.blend@ for writing: No such file or directory

I don't have a blender_files directory. I think this one follows from the comment above about creating folders

Reply:
Also, the created .blend file shows up in my git. Suggest to add the line to the `.gitignore` file:

*.blend

to remove tracking of all blend files without explicitly being added.